### PR TITLE
Doc edits

### PR DESCRIPTION
**Pull request by crandmck**
## Changes in this pull request

Edits and reorganization of documentation.  

I have a few open questions: 

- **Installation**: _What can we say about OS support?  Mac only?_ I suspect it is meant to work on Windows and Linux, but Windows always requires extra work (i.e. using WSL2).  If it hasn't been tested on other OSes then we should say so.
- **Testing locally in the Git directory section**:  I don't understand what this means "If you just want to do local testing builds..."   Does this mean local build/testing of the tool itself?  If this is for building the tool from source, I would clarify a bit and move this section to a more appropriate location.
- This repo is referenced a number of times, but it's private and won't be visible to anyone outside the c2pa org: <https://github.com/c2pa-org/testing-private/tree/main/cert-generation>.  Either don't mention it, replace with an alternative, or ask that it be made public ASAP.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,57 @@
...
 
-Each C2PA tool or service will have its own unique behavior and technology stack. Therefore, this tool is a framework for generating malicious images for testing in a manner that can be customized for specific targets. It is expected that the user of the tool will create customized attack files for their specific environment. The files provided with this tool are just examples for initial experimentation.
+This tool facilitates security testing early in the development cycle of Content Credentials applications. For such applications, often the first step in processing an asset is to validate the signature and reject any whose public key is not from a trusted CA. That said, it is still good to test the parsers since hackers could find ways to get unexpected data into C2PA manfiest fields. In addition, it is conceivable that a Content Credentials application could parse manifest data without validating the certificate. Therefore, it is critical that the application safely handles unexpected input.
```
Comment (scouten-adobe) [inline README.md:22]: Typo: manfiest should be manifest.

File: README.md [hunk 2]
```diff
@@ -113,159 +98,61 @@ The tool works with the following types of asset files (also referred to as _ass
...
+These examples create output in the `sample_out` directory.
 
-#### Method 1: Direct substitutions
+### Inject into the author field using direct substitution
 
-The simplest approach is to directly inject into the compiled manifest before signing. This approach is only supported for a few common fields because there are too many possible manifest fields to make them all available from a command line. The advantage of this approach is that it is an easy way to start testing the most common fields without understanding JSON manifests. The currently available options are the title, author, and claim generator. More fields will be added as the tool matures. The injections in this approach are done after the JSON file has been imported and turned into a manifest structure in memory. Therefore, any type of character can be injected via this method.
+The following command is an example of using direct substituion. For a general explanation of using direct substitution, see [Using c2pa-attacks](./docs/usage.md#direct-substitution).
```
Comment (scouten-adobe) [inline README.md:109]: Typo: substituion -> should be substitution.

File: README.md [hunk 3]
```diff
@@ -113,159 +98,61 @@ The tool works with the following types of asset files (also referred to as _ass
 
 NOTE: Quicktime (`.mov`) format is not yet fully supported.
 
-## Usage
```
Comment (scouten-adobe) [inline README.md:116]: Important caveat for my review: The diffs are really hard to follow here, so I am not checking to ensure that all of Peleus' original content is accounted for somewhere. I'm only reading / proofreading the new content.

File: README.md [hunk 4]
```diff
@@ -113,159 +98,61 @@ The tool works with the following types of asset files (also referred to as _ass
...
+## Testing Certificate Authority fields
 
-- `private_key`
-- `sign_cert`
+Security researchers can create self-signed certificate authorities and leaf certificates based on them. The C2PA project provides tools for generating certificates using OpenSSL in this repository: <https://github.com/c2pa-org/testing-private/tree/main/cert-generation>.
 
-If you are using a signing algorithm other than the default `es256`, specify it in the manifest definition field `alg` with one of the following values:
+Within this project, the sample directory has a certificate with some unexpected values in common fields that can be used for signing. These certificates are based on a self-signed CA so any tools that validate against a trusted CA list won't accept them. However, you can use them to ensure that your certificate parsers can handle unexpected characters. They do not represent a complrehensive attack suite but it is a place to start.
```
Comment (scouten-adobe) [inline README.md:150]: Typo: complrehensive -> comprehensive.

File: attacks/README.md [hunk 5]
```diff
@@ -1,28 +1,34 @@
...
+
+This directory contains the following files:
+
+- `xss.attack`: A few simple XSS injections.
+- `ten_thousand_characters.attack`: A 10,000 character string to test for buffer overflows.
+- `sql_injection.attack`: A few simple SQL injection techniques. 
+- `rendering.attack`: A simple "Hello World" attack that contains newlines to inspire testing rendering techniques that split the UI display with injected newlines.
+- `special_characters.attack`: A selection of special characters as a way to test special character handling in the UI.
+
+ NOTE: The `.gitattributes` file in this directory maps the `.attack` filename extension to `binary` so that GitHub will persist binary characters in those files.
```
Comment (scouten-adobe) [inline attacks/README.md:17]: Extra leading space here

File: attacks/README.md [hunk 6]
```diff
@@ -1,28 +1,34 @@
...
+
+## Generating long strings
+
+To inject really long strings for triggering buffer overflows, it is easy to generate strings with some simple command lines. For exampl,e this command generates 100 'a' characters in the file `output_file.attack`:
```
Comment (scouten-adobe) [inline attacks/README.md:21]: Typo: "exampl,e" -> "example,"

**Review by scouten-adobe — APPROVED** [review]
Aside from a couple of typos, this looks really solid. I do want to see @puhley's feedback, though, as well.

File: README.md (hunk 3 above)
Comment (crandmck) [reply README.md:116]: Yes, I agree the diffs are hard to follow.  Perhaps the best option is to view the files directly in the branch: https://github.com/contentauth/c2pa-attacks/blob/doc-edits/README.md.  This won't directly tell you if everything is there vs. the original, but you can more easily read it to see if it's complete and compare it side-by-side with the original.   Certainly, my intent is to keep all the original information, but make it more clear and concise.

**Comment by crandmck** [comment]
Thanks for catching the typos, @scouten-adobe.  

I have now installed a spell-checker extension to my VS Code to help avoid these kinds of embarrassing typos in the future!

**Review by puhley — APPROVED** [review]
